### PR TITLE
Multiplayer help links and base page

### PR DESCRIPTION
**Pull request by ganicke**
Fixes the help links for the multiplayer UI href elements. Also, a base `multiplayer.md` page is added as a place holder for `@extends`.

### CODE REVIEW

File: common-docs/multiplayer.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+## Host a game #host-game
+
+You host a multiplayer game by using the **Share** button at the top of the editor screen.
+
+Choose **Host a multiplayer game** instead of creating a share link to your project.
+
+Once you choose to host your game, a new window opens with your game's join code. You share this code with the other players so that they can join your game. You invite other players to join your game in one of three ways:
+
+* Give them the six-digit game code and when they navigate to the [MakeCode Game Lobby](https://aka.ms/a9b), or https://aka.ms/a9b, they can enter the code to join.
+* Scan the QR code with your mobile phone and send them a message with the join link.
```
Comment (eanders-ms) [inline common-docs/multiplayer.md:22]:
```suggestion
* Have your friend scan the QR code with their mobile phone to join the game.
```
The QR code will direct-join the person scanning it.

File: common-docs/multiplayer.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+## Host a game #host-game
+
+You host a multiplayer game by using the **Share** button at the top of the editor screen.
+
+Choose **Host a multiplayer game** instead of creating a share link to your project.
+
+Once you choose to host your game, a new window opens with your game's join code. You share this code with the other players so that they can join your game. You invite other players to join your game in one of three ways:
+
+* Give them the six-digit game code and when they navigate to the [MakeCode Game Lobby](https://aka.ms/a9b), or https://aka.ms/a9b, they can enter the code to join.
```
Comment (eanders-ms) [inline common-docs/multiplayer.md:21]:
```suggestion
* Give them the six-digit game code and when they navigate to the [MakeCode Game Lobby](https://aka.ms/a9), or https://aka.ms/a9, they can enter the code to join.
```
The `b` suffix points to `/beta`. We want to route to the live endpoint, so just `aka.ms/a9` ( no `b`).

File: common-docs/multiplayer.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+Once your game starts, the host's view of the game displays and you can begin game play or wait for other players to join.
+
+## Join a game #join-game
+
+To join a multiplayer game hosted by another player, you can go to the [MakeCode Game Lobby](https://aka.ms/a9b), or https://aka.ms/a9b, and enter the game's join code that you received from the game host.
+
+After you press the **Enter** button, your game view will display and you can begin playing. If you received a full join URL instead, such as https://aka.ms/a9b?join=F98987, this will take you directly to your player's game window.
+
+## Multiplayer game safety #safety
```
Comment (eanders-ms) [inline common-docs/multiplayer.md:35]: @kiki-lee

File: common-docs/multiplayer.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+The "online multiplayer" mode is a user-controlled, self-contained sharing system that allows creators to play their games with friends in a safe environment that does not allow exchange of personal information.
+
+Users are expected to must share their game join code with classmates directly, as it is neither stored nor discoverable through the game galleries.
```
Comment (eanders-ms) [inline common-docs/multiplayer.md:39]:
```suggestion
Users are expected to share their game join code with classmates directly, as it is neither stored nor discoverable through the game galleries.
```

File: common-docs/multiplayer.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+
+Once inside an online multiplayer session, MakeCode does not provide tools for messaging any players whether joined to a host's game or not. Also, we do not provide any way to share personal contact information that might lead to opening chat session communication outside of the tool, including names, usernames, and/or profile images.
```
Comment (eanders-ms) [inline common-docs/multiplayer.md:41]: This paragraph feels technical/wordy, like a rough draft. Maybe this suggestion will help angle it toward refinement:
```
In multiplayer Arcade games, player identities are protected. Communication between players is limited to a fixed set of emojis. Text chat, voice chat, etc. are not supported.
```

Comment (ganicke) [reply common-docs/multiplayer.md:41]: Yeah, we could soften that a bit. I didn't do much with that section that was copied in.

**Review by abchatra — APPROVED** [review]

**Comment by abchatra** [comment]
I am merging this to produce the build. We can add other stuff later.

**Pull request merged**